src/app/components/crypto-assets/crypto-currency-asset/crypto-currency-asset-item.layout.tsx

```diff
@@ -57,10 +58,14 @@ export function CryptoCurrencyAssetItemLayout({
         }
         captionRight={
           !rightElement && (
-            <>
-              <Caption>{balance.amount.toNumber() > 0 && address ? usdBalance : null}</Caption>
-              {additionalUsdBalanceInfo}
-            </>
+            <Caption>
```

Comment:
Thanks for fixing, this was prob me breaking it implementing the `ItemInteractive`. I remember encountering this component again recently, questioning if it is really a `layout` component now when it is just rendering an `ItemInteractive` that that wraps the `ItemLayout`? Maybe we don't even need to have the props passed to render here?